docs/getting-started/ios/index.md

```diff
@@ -1,5 +1,75 @@
 
 # Getting started for iOS developers
 
-This page describes how to get started building an application with Warp components.
++To get started, you need to install the `warp-ds/warp-ios` package
 
+## 1. Integrate Warp
+
+A guide on how to integrate Warp into your project.
+
+### Swift Package Manager
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/warp-ds/warp-ios.git")
+]
+```
+
+## 2. Use Warp components
+
+Brand: to set the colortheme for the market you can specify what brand the app is using, then in the market cell is set the tintcolor is defined.
+  
+var brand: String = "" {
+    didSet {
+        if brand == "Tori" {
+             iconImageView.tintColor = UIColor(red: 247/255, green: 81/255, blue: 89/255, alpha: 1)
+
+        }
+        else if brand == "Finn" {
+           iconImageView.tintColor = UIColor(red: 6/255, green: 190/255, blue: 251/255, alpha: 1)
+
+        }
```

Comment:
Hmm is it not a better way to switch the branding, than to use if else sentences and hardcoding the color?